.github/workflows/publish_client_android_release_candidate.yml

```
@@ -0,0 +1,68 @@
+name: Publish Client Android Release Candidate
```

I guess at this point it's the release, not a candidate anymore
```suggestion
name: Publish Client Android
```